val1, val2 = 3, 0
try:
    a = val1/val2
except Exception as e:
    print(e)
